roles/openshift_health_checker/openshift_checks/logging/podlogs.py

```
@@ -0,0 +1,174 @@
+            self.execute_module,
+            self.logging_namespace,
+            "kibana",
+            task_vars,
+        )
+
+        if error:
+            return {"failed": True, "changed": False, "msg": error}
+
+        running_kibana_pods = self.running_pods(pods)
+        if not len(running_kibana_pods):
+            msg = ('No Kibana pods were found to be in the "Running" state.'
+                   'At least one Kibana pod is required in order to perform this check.')
+            return {"failed": True, "changed": False, "msg": msg}
+
+        # get all Elasticsearch pods
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default=self.logging_namespace)
+        pods, error = super(PodLogQueryTime, self).get_pods_for_component(
+            self.execute_module,
+            self.logging_namespace,
+            "es",
+            task_vars,
+        )
+
+        if error:
+            return {"failed": True, "changed": False, "msg": error}
+
+        running_es_pods = self.running_pods(pods)
+        if not len(running_es_pods):
+            msg = ('No Elasticsearch pods were found to be in the "Running" state.'
+                   'At least one Elasticsearch pod is required in order to perform this check.')
+            return {"failed": True, "changed": False, "msg": msg}
+
+        error = self.check_log_delay(running_kibana_pods[0], running_es_pods[0], pod_log_timeout, task_vars)
+        if error:
+            return {"failed": True, "msg": error}
+
+        return {}
+
+    def check_log_delay(self, kibana_pod, es_pod, pod_log_timeout, task_vars):
+        """Check delay of logs and return any errors accordingly"""
+        error = None
+
+        try:
+            uid = self.curl_kibana_with_uuid(kibana_pod, task_vars)
+            self.wait_until_cmd_or_err(es_pod, uid, pod_log_timeout, task_vars)
+        except OpenShiftCheckException as error:
```

Don't catch this, just let it bubble up.